extra/1-radio-stations.js

```diff
@@ -25,6 +29,14 @@
  * - Return only the frequencies that are radio stations.
  */
 // `getStations` goes here
+function getStations() {
+  if (isRadioStation()) {
+    return true
+  } else {
+    return getAvailableStations()
```

Comment:
This function technically passes the test, however, getAvailableStations is a test function that we shouldn't be using, it does the same thing that we are trying to write our own function for! Hopefully you understand how to do this after we went through it last week but feel free to ask any questions about it if not :)